pytorch_lightning/callbacks/progress/base.py

```diff
@@ -153,6 +149,8 @@ def print(self, *args: Any, **kwargs: Any) -> None:
 
     def setup(self, trainer: "pl.Trainer", pl_module: "pl.LightningModule", stage: Optional[str] = None) -> None:
         self._trainer = trainer
+        # if not trainer.is_global_zero:
+        #     self.disable()
```

Comment:
Are you having trouble here? Is there an issue with tests failing?

Reply:
yes :/ the issue is https://github.com/PyTorchLightning/pytorch-lightning/pull/11377#issuecomment-1009489813

test failing with https://gist.github.com/daniellepintz/205b9b05750fa212534b66a5e8ac1f08

Reply:
I am quite confused because this test shouldnt even interact with the progress bar at all: https://github.com/PyTorchLightning/pytorch-lightning/blob/18bbb39eef77c8c3d19789b1067164e962640c93/tests/checkpointing/test_model_checkpoint.py#L154

Reply:
I have identified the problem is due to the test I added `test_tqdm_progress_bar_disabled_when_not_rank_zero`, but I am still trying to figure out why